app/models/loan_response.rb

```diff
@@ -38,7 +40,14 @@ def hash_data
   end
 
   def embeddable_media
-    embeddable_media_id.present? ? EmbeddableMedia.find(embeddable_media_id) : nil
+    if embeddable_media_id.present?
+      result = EmbeddableMedia.find_safe(embeddable_media_id)
+    end
+    unless result
+      result = EmbeddableMedia.where(owner_id: @owner.id, owner_type: @owner.class.name,
+        owner_attribute: @custom_field.id).first
+    end
+    result
```

Comment:
This logic was to adapt between previously migrated data which only had a forward reference from the question response data, and the changed model where there is a back reference from the embeddable media instance.  The model change was needed so that access policy checks could be enforced with just the embeddable media data, instead of needing to pass in external context.